app/services/people/retrieve_image.rb

```diff
@@ -3,6 +3,7 @@
 module People
   class RetrieveImage
     def self.call(person)
+      return false unless person.latest_nomis_booking_id
```

Comment:
I am very new to this, so may be talking nonsense, but should this line not be placed after line 7 so that if there's an existing image attachment that gets returned irrespective of the missing `latest_nomis_booking_id` ... not sure if that's a valid scenario even?

Reply:
That's a good point, I'll make the change.

(It does not change in case you got only one profile, since `latest_nomis_booking_id` is gonna be the same for the first and subsequent calls.)